### PR TITLE
blueprint for languages sub generator

### PR DESCRIPTION
**Pull request by pascalgrimaud**
Related to https://github.com/jhipster/jhipster-vuejs/issues/132

This is needed in Vue.js blueprint.
cc @sahbi-ktifa 

Now, I understand the amount of work you did in blueprint / refactoring yeoman. I spent days on this!
That's why I assigned to you @murdos, if you can help in review.

_____

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/languages/index.js [hunk 1]
```diff
@@ -78,69 +80,80 @@ module.exports = class extends BaseGenerator {
                 }
             });
         }
+        const blueprint = this.options.blueprint || this.configOptions.blueprint || this.config.get('blueprint');
+        // use global variable since getters dont have access to instance property
+        if (!opts.fromBlueprint) {
+            useBlueprint = this.composeBlueprint(blueprint, 'languages', {
+                'skip-install': this.options['skip-install'],
```
Comment (murdos) [inline generators/languages/index.js]: I'm pretty sure it's best to pass the same options to the blueprint subgenerator as the ones the jhipster subgenerator has received: https://github.com/jhipster/generator-jhipster/blob/master/generators/generator-base.js#L2691
So I would also add: `'skip-server'`, `'skip-client'`, `langages` and probably `debug` too.

Comment (pascalgrimaud) [reply generators/languages/index.js]: thx, it's done

**Comment by pascalgrimaud** [comment]
My bad! I didn't saw there are some tests for blueprint.
Thanks for reviewing @murdos !

**Comment by deepu105** [comment]
You have no idea how happy I'm to see you guys taking care of all these :)

Thanks & Regards,
Deepu

**Comment by pascalgrimaud** [comment]
Thanks @deepu105 ! I'm happy too to learn new stuff :heart:

**Comment by pascalgrimaud** [comment]
if you're ok, I'll merge this tonight so we can test it in jhipster-vuejs

**Comment by murdos** [comment]
Sorry, I was busy and did not see your comment.
You did the right thing, there were nothing blocking the merge of this PR!

Just for the record, there's just one thing that was bothering me : the fake blueprint in the unit test is not doing anything special, so we're just testing that the jhipster generator is executing. So this could be improved.

**Comment by pascalgrimaud** [comment]
agree @murdos, I saw a lot of tests can be improved

**Pull request merged**